modules/fieldlinessequence/rendering/renderablefieldlinessequence.cpp

```diff
@@ -171,7 +172,7 @@ namespace {
     };
 
     constexpr openspace::properties::Property::PropertyInfo FlowSpeedInfo = {
-        "Speed",
+        "FlowSpeed",
```

Comment:
This is a breaking change that we need to make sure that none of our assets are using the old version and it needs to be in the breaking changes list of the internal wiki

Reply:
This was used in Juice profile for GanymedeMagnetosphere together with DomainEnabled and FlowEnabled.
Name change + that DomainEnabled and FlowEnabled is noted in release notes, and edited in profile.